components/TaggedValue/TaggedValue.jsx

```diff
@@ -0,0 +1,30 @@
+require('./TaggedValue.scss')
+
+import React from 'react'
+import classNames from 'classnames'
+
+const TaggedValue = ({title,subText,style,count}) => {  
+  const taggedValueClass = classNames(
+    'TaggedValue',
+    { [`${style}`]: style,
```

Comment:
Did you try `[style] : true`? It should work. And you can set a default value, which you can define in TaggedValue.scss, for style property. Or Am I missing something here?